Arguments: 
  C:\Program Files\nodejs\node.exe C:\Users\harsh\AppData\Roaming\npm\node_modules\yarn\bin\yarn.js

PATH: 
  C:\Users\harsh\AppData\Local\Programs\Python\Python311\Scripts\;C:\Users\harsh\AppData\Local\Programs\Python\Python311\;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\MinGW\bin;C:\Program Files\Java\jdk-17.0.3.1\bin;C:\Program Files\kotlinc\bin;C:\Program Files\Git\cmd;C:\Program Files\MongoDB\Server\6.0\bin;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Users\harsh\AppData\Local\Microsoft\WindowsApps;C:\Users\harsh\AppData\Local\Programs\Microsoft VS Code\bin;C:\ffmpeg\bin;C:\Program Files\kotlinc\bin;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin;;C:\Users\harsh\AppData\Roaming\npm;C:\Users\harsh\OneDrive\Documents\PROJECTS\ai-saas;

Yarn version: 
  1.22.19

Node version: 
  18.16.1

Platform: 
  win32 x64

Trace: 
  Error: connect EHOSTUNREACH 2606:4700:9ad3:80e2:e4ad:1a:6810:23:443
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16)

npm manifest: 
  {
    "type": "module",
    "scripts": {
      "start": "nodemon index.js"
    },
    "dependencies": {
      "@prisma/client": "^4.14.0",
      "cors": "^2.8.5",
      "crypto": "^1.0.1",
      "dotenv": "^16.0.3",
      "express": "^4.18.2",
      "multer": "^1.4.5-lts.1",
      "nodemon": "^2.0.20",
      "prisma": "^4.14.0",
      "socket.io": "^4.6.1",
      "ts-node": "^10.9.1",
      "typescript": "^4.8.4"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
